src/components/visual/transaction-list-item/transaction-list-item.scss

```diff
@@ -99,6 +111,18 @@
   .transaction-amount-label {
     @apply mvx:opacity-70 mvx:tracking-tight;
   }
+
+  @media (min-width:375px) {
```

Comment:
These two media queries sohuld be directly under `.transactions-amount`, positioned above `&.amount-positive`, and it shouldn't required you to write the class anymore either, same as the media queries from line 44.